test/xpu/test_conv.py

```diff
@@ -1264,6 +1264,17 @@ def test_channels_last_ouput_stride(self, device, dtype):
             # input NHWC, output NHWC
             assert_size_stride(out, (2, 512, 7, 7), (25088, 1, 3584, 512))
 
+    @onlyXPU
+    def test_mkldnn_allow_tf32_get_set(self, device):
```

Comment:
Is `device` a useless variable here?

Reply:
Yes, it is now removed in newest commit, thanks.